queries/ne.jinja2

```diff
@@ -268,7 +268,7 @@ FROM
 WHERE
 
   {{ bounds['line']|bbox_filter('the_geom', 3857) }}
-  AND scalerank <= {{ zoom }}
+  AND {{ zoom }} >= 5 AND scalerank <= {{ zoom }}
```

Comment:
does `between` fit here?

    {{ zoom }} between 5 and scalerank

But, if possible it might be better to just guard the whole query around a conditional to prevent it from executing at all if the zoom is < 5.